src/Libraries/CoreNodeModelsWpf/NodeViewCustomizations/DummyNode.cs

```diff
@@ -14,13 +15,18 @@ public void CustomizeView(DummyNode model, Dynamo.Controls.NodeView nodeView)
             if (model.NodeNature == DummyNode.Nature.Unresolved)
                 fileName = "MissingNode.png";
 
+            model.State = ElementState.Warning;
```

Comment:
why are we modifying this?

Reply:
The DummyNode is a bit weird, it doesn't seem to want to recognise it's in a warning state for some reason. 
It only appears due to an unresolved node, so thought I'd just force it.